contrib/prepare_release.sh

```diff
@@ -83,19 +83,24 @@ for plat in x86_64 i686 arm ppc64le; do
     s3://julialang/bin/linux/$platshort/$majmin/
   aws s3 cp --acl public-read julia-$majmin-latest-linux-$plat.tar.gz \
     s3://julialang/bin/linux/$platshort/$majmin/
+  curl -X PURGE -L "https://julialang-s3.julialang.org/bin/linux/$platshort/$majmin/julia-$majmin-latest-linux-$plat.tar.gz"
 done
 aws s3 cp --acl public-read "julia-$version-osx10.7 .dmg" \
   s3://julialang/bin/osx/x64/$majmin/
 aws s3 cp --acl public-read "julia-$majmin-latest-osx10.7 .dmg" \
   s3://julialang/bin/osx/x64/$majmin/
+curl -X PURGE -L "https://julialang-s3.julialang.org/bin/osx/x64/$majmin/julia-$majmin-latest-osx10.7 .dmg"
```

Comment:
maybe have both with the space and with the plus, in case people are using both as links and fastly considers them different? s3 normalizes them to the same file but fastly might not